cysync/src/store/provider/syncProvider/executors/history.ts

```diff
@@ -288,8 +292,8 @@ export const processResponses = async (
         new BigNumber(ele.gasUsed || 0)
       );
 
-      const fromAddr = formatEthAddress(ele.from);
-      const toAddr = formatEthAddress(ele.to);
+      const fromAddr = ele.from;
+      const toAddr = ele.to;
```

Comment:
These addresses and the variable `address` should be in the same format and case for this to work.

Reply:
can we keep `address` to lower case?

Reply:
Yes